### PR TITLE
tests/main/confinement-classic: enable the test on Fedora

### PR DESCRIPTION
**Pull request by bboozzoo**
The snap-exec failure has been addressed with
563795fd8d628c64da4019aee3f5c6a845eb0fe7. Enable the test on Fedora and recreate
the manual steps that need to be done in order to support classic confinement.

### CODE REVIEW

**Comment by bboozzoo** [comment]
These failed: 
```
2018-01-03 15:53:47 Failed tasks: 5
    - linode:debian-9-64:tests/main/confinement-classic
    - linode:fedora-26-64:tests/main/confinement-classic
    - linode:ubuntu-14.04-64:tests/main/confinement-classic
    - linode:ubuntu-16.04-32:tests/main/confinement-classic
    - linode:ubuntu-16.04-64:tests/main/confinement-classic
```

There are 2 problems right now. The first one is `test-snapd-hello-classic` snap is only in edge channel for amd64, @mvo5 can I release it to stable/beta/candidate now? The second issue is missing snap for `i386`, which I have just now built and pushed, but it needs approval (it's a classic snap after all). Once this is approved, I'll release it to the edege/beta/candidate/stable as well and we should get a green build.

**Comment by bboozzoo** [comment]
Snaps are available now, restarted the build

**Comment by bboozzoo** [comment]
The build is finally green. Ready for re-review.

File: tests/main/confinement-classic/task.yaml [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
-    run_install() {
-        make -C test-snapd-hello-classic clean
-        make -C test-snapd-hello-classic
-        snap install "$@" --dangerous ./test-snapd-hello-classic/test-snapd-hello-classic_0.1_*.snap
-    }
+execute: |
+    . $TESTSLIB/dirs.sh
 
-    run_install --classic
+    snap install --classic test-snapd-hello-classic
```
Comment (zyga) [inline tests/main/confinement-classic/task.yaml]: Super, thank you for doing this!

**Pull request merged**